troupon/static/scss/partials/_resets.scss

```diff
@@ -0,0 +1,60 @@
+
+/* ==========================================================================
+Resets (overrides for Bootstrap and browser-defaults)
+========================================================================== */
+
+body {
+  appearance: none;
+}
+
+h1, h2, h3, h4 {
+  margin: 0px 0px 10px 0px;
+}
+
+ul, ol {
+  padding: 0px;

```

Comment:
Properties should be ordered margin, padding<br>`0px` should be written without units as `0`